src/components/Molecules/Membership/Form/Form.js

```diff
@@ -178,7 +178,7 @@ const Signup = ({
         </FormFieldset>
         <Button type="submit">Donate</Button>
         {errorMsg && (
-          <Error>
+          <Error as="p">
```

Comment:
You can use `tag="p"` Error is Text component.

Reply:
Done